### ISSUE TITLE
tooltip in a radar chart is not displayed.

### ISSUE THREAD

**Issue by squll1** [recharts/recharts#146]
```js
const {Radar, RadarChart, PolarGrid, Legend,
         PolarAngleAxis, PolarRadiusAxis, Tooltip} = Recharts;

const data = [
    { subject: 'Math', A: 120, B: 110, fullMark: 150 },
    { subject: 'Chinese', A: 98, B: 130, fullMark: 150 },
    { subject: 'English', A: 86, B: 130, fullMark: 150 },
    { subject: 'Geography', A: 99, B: 100, fullMark: 150 },
    { subject: 'Physics', A: 85, B: 90, fullMark: 150 },
    { subject: 'History', A: 65, B: 85, fullMark: 150 },
];

const TwoLevelPieChart = React.createClass({
	render () {
  	return (
    	<RadarChart cx={300} cy={250} outerRadius={150} width={600} height={500} data={data}>
          <Radar name="Mike" dataKey="A" stroke="#8884d8" fill="#8884d8" fillOpacity={0.6}/>
          <Radar name="Lily" dataKey="B" stroke="#82ca9d" fill="#82ca9d" fillOpacity={0.6}/>
          <PolarGrid />
	  <Tooltip/>
          <Legend />
          <PolarAngleAxis dataKey="subject" />
          <PolarRadiusAxis angle={30} domain={[0, 150]}/>
        </RadarChart>
    );
  }
})

ReactDOM.render(
  <TwoLevelPieChart />,
  document.getElementById('container')
);

```
Add Tooltip components , but it does not appear in the tooltip

**Comment by xile611** [comment]
@squll1 RadarChart don't support tooltip now.

**Comment by xile611** [comment]
Fixed in version 0.12.6.
